### DISCUSSION TITLE
We are experiencing some technical difficulties with the create new post function.

### DISCUSSION THREAD

**Discussion by novium2** [community/community#142455] [Announcements]
We are experiencing some technical difficulties with the creation of new posts. We are working to restore the ability to create new posts as quickly as possible. Thank you!

**Comment by novium2** [comment]
This has now been fixed. Thank you for your patience!

**Reply by surekha09-crypto** [reply]
thanks

**Reply by Meerareemsalem** [reply]
> This has now been fixed. Thank you for your patience!

What been fixed nothing

**Reply by Meerareemsalem** [reply]
What been fixed

**Comment by vau5vau** [comment]
![Screenshot_20250325-165028](https://github.com/user-attachments/assets/80ab510a-f14a-40c1-86d2-274efd359465)
Помогите У меня 13.03.2025 г , в 11:48 03 завис платеж чек перевода у меня есть
